tests/messages_tests/test_fallback.py

```diff
@@ -42,6 +42,12 @@ def stored_messages_count(self, storage, response):
             self.stored_session_messages_count(storage, response)
         )
 
+    def test_repr(self):
+        storage = self.get_storage()
+        self.assertEqual(
+            repr(storage),
+            f'FallbackStorage(storages = {storage.storages!r})')
```

Comment:
Please use hanging indentation. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).